project-base/build-dev.xml

```diff
@@ -16,12 +16,13 @@
     <target name="standards" depends="phplint,ecs,phpstan,twig-lint,yaml-lint,eslint-check" description="Checks coding standards."/>
     <target name="standards-diff" depends="phplint-diff,ecs-diff,phpstan,twig-lint-diff,yaml-lint,eslint-check-diff" description="Checks coding standards on changed files."/>
 
-    <target name="test-db-demo" depends="clean,redis-check,clean-redis,test-db-wipe-public-schema,test-db-import-basic-structure,test-db-migrations,test-create-domains-data,test-db-fixtures-demo,test-generate-friendly-urls,test-load-plugin-demo-data,test-replace-domains-urls" description="Drops all data in test database and creates a new one with demo data."/>
+    <target name="test-db-demo" depends="clean,redis-check,clean-redis,test-db-wipe-public-schema,test-db-import-basic-structure,test-db-migrations,test-create-domains-data,test-db-fixtures-demo,test-generate-friendly-urls,test-load-plugin-demo-data,test-replace-domains-urls,test-product-search-recreate-structure,test-product-search-export-products" description="Drops all data in test database and creates a new one with demo data."/>
```

Comment:
please update also description about elasticsearch indices and filled data.

Reply:
BUT.
product-search-recreate-structure,product-search-export-products is contained in `build-demo-dev` target
db-demo does not contain stuff with elastic work. Please look also into db-demo and unify both `db-demo` and `test-db-demo` targets.

Reply:
There is difference between `test-db-demo` and `db-demo` anyway. So probably we should create a issue to unify that targets.

Reply:
hmm OK, then please update also docs 
`docs/introduction/console-commands-for-application-management-phing-targets.md:118`
`docs/introduction/running-acceptance-tests.md:72`